src/firewheel/lib/minimega/api.py

```diff
@@ -49,11 +49,17 @@ def __init__(self, mm_base=None, timeout=120, skip_retry=False):
         self.mm_base = mm_base
         self.mm_socket = os.path.join(self.mm_base, "minimega")
 
+        try:
+            namespace = config["minimega"]["namespace"]
+        except KeyError:
+            self.log.warning("minimega namespace not set, using default")
+            namespace = None
+
```

Comment:
To avoid misleading errors in the case where the `KeyError` is raised because `minimega` is somehow missing from the `config` option instead of `namespace`, could we substitute the following equivalent block?

```suggestion
        if (namespace := config["minimega"].get("namespace")) is None:
            self.log.warning("minimega namespace not set, using default")
```